### PR TITLE
[quant][2.1] fix formatting for the tutorial

### PR DESCRIPTION
**Pull request by jerryzh168**
Summary:
.

Test Plan:
visual inspection of generated docs

Reviewers:

Subscribers:

Tasks:

Tags:

Fixes #ISSUE_NUMBER

## Checklist

- [ ] The issue that is being fixed is referred in the description (see above "Fixes #ISSUE_NUMBER")
- [ ] Only one issue is addressed in this pull request
- [ ] Labels from the issue that this PR is fixing are added to this pull request
- [ ] No unnecessary issues are included into this pull request.

### CODE REVIEW

File: prototype_source/pt2e_quant_ptq_static.rst [hunk 1]
```diff
@@ -457,7 +454,9 @@ we offer in the long term might change based on feedback from PyTorch users.
        out_fp32, out_scale, out_zero_point, out_quant_min, out_quant_max, torch.int8)
        return out_i8
      
-* Reference Quantized Model Representation (WIP, expected to be ready at end of August): we have special representation for selected ops (for example, quantized linear), other ops are represented as (``dq -> float32_op -> q``), and ``q/dq`` are decomposed into more primitive operators.
+* Reference Quantized Model Representation (available in nightlies)
```
Comment (svekars) [inline prototype_source/pt2e_quant_ptq_static.rst:457]:
```suggestion
* Reference Quantized Model Representation (available in the nightly build)
```

File: prototype_source/pt2e_quant_ptq_static.rst [hunk 2]
```diff
@@ -457,7 +454,9 @@ we offer in the long term might change based on feedback from PyTorch users.
...
+
+  We will have special representation for selected ops (for example, quantized linear), other ops are represented as (``dq -> float32_op -> q``), and ``q/dq`` are decomposed into more primitive operators.
```
Comment (svekars) [inline prototype_source/pt2e_quant_ptq_static.rst:459]:
```suggestion
  We will have a special representation for selected ops, for example, quantized linear. Other ops are represented as ``dq -> float32_op -> q`` and ``q/dq`` are decomposed into more primitive operators.
```

**Review by svekars — APPROVED** [review]
LGTM

**Comment by jerryzh168** [comment]
Hi @svekars could you merge this?

**Pull request merged**